client/js/components/ParkFeatureListItem.jsx

```diff
@@ -4,18 +4,23 @@ import icons from '../utils/icons.json';
 
 
 export default class ParkFeatureListItem extends React.Component {
+    onClick() {
+        this.props.selectFeature(this.props.id);
+        this.props.slideUp();
```

Comment:
Hmm, so I was initially going to do that but 

* ParkPage defines selectFeature
* Container defines slideUp
* Container is a subcomponent of ParkPage

So the parent would have to reach into its child's properties in order to invoke slideUp. I think this is anti-pattern for react?